README.md

```diff
@@ -30,7 +38,7 @@ app_keys = {
 
 access_keys = {
   'jonathankoren' : {	    # a specific account. You can change this, but
```

Comment:
How I assumed it worked at first is that you could have as many key/value pairs as components of `access_keys` as you wanted, and the script would process all of them. It seems like it would be easy to loop through all the keys in `access_keys` (the only reason why I didn't submit a PR to do that is that I don't know that much Python :)